{"uid":"6cc7b42158310caf","name":"test_async_replication_failover_in_secondary_cluster","fullName":"tests.integration.ha_tests.test_async_replication#test_async_replication_failover_in_secondary_cluster","historyId":"906add62d7ba7dfa7a2694f87ce2425e","time":{},"description":"Test that async replication fails back correctly.","descriptionHtml":"<p>Test that async replication fails back correctly.</p>\n","status":"unknown","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"@pytest.mark.skipif(False, reason='Requires juju 3')"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_async_replication"},{"name":"host","value":"fv-az1148-73"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_async_replication"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":1,"unknown":3,"total":4},"items":[{"uid":"99844603cf0f7ae","reportUrl":"../72//#testresult/99844603cf0f7ae","status":"unknown","time":{}},{"uid":"1d8e352d52ad48dc","reportUrl":"../71//#testresult/1d8e352d52ad48dc","status":"unknown","time":{}},{"uid":"c5d914462aec1243","reportUrl":"../70//#testresult/c5d914462aec1243","status":"passed","time":{"start":1717814708927,"stop":1717814797363,"duration":88436}}]},"tags":["asyncio","abort_on_fail","@pytest.mark.skipif(False, reason='Requires juju 3')","group(1)"]},"source":"6cc7b42158310caf.json","parameterValues":[]}